client/components/product-details.jsx

```diff
@@ -12,11 +12,9 @@ export default class ProductDetails extends React.Component {
   fixPrice(price) {
     const fixedPrice = this.state.product.price.toFixed();
     if (fixedPrice.length === 3) {
-      price = '$' + fixedPrice[0] + '.' + fixedPrice[1] + fixedPrice[2];
-      return price;
+      return '$' + fixedPrice[0] + '.' + fixedPrice[1] + fixedPrice[2];
     } else if (fixedPrice.length === 4) {
-      price = '$' + fixedPrice[0] + fixedPrice[1] + '.' + fixedPrice[2] + fixedPrice[3];
-      return price;
+      return '$' + fixedPrice[0] + fixedPrice[1] + '.' + fixedPrice[2] + fixedPrice[3];
```

Comment:
Looks like you are doing a lot of formatting of prices. What if you put a dedicated function in its own module and just used that everywhere?